ui/package.json

```diff
@@ -42,6 +42,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.8",
+    "@paciolan/remote-component": "^2.10.2",
```

Comment:
Needs to go into `master`?

Reply:
@vineethvanga18 how is the MeshMap user preference going to be remotely loaded if this utility component isn't in `master`?

Reply:
Yes these should be in master.
And I guess @utkarsh-pro already pushed these to master

Reply:
@vineethvanga18 ready to check-in MeshMap user pref into Meshery Extensions repo?

Reply:
Will do by today @leecalcote